src/zero/module/wl.clj

```diff
@@ -32,9 +33,10 @@
   "Use transaction TX to update _WORKLOAD statuses."
   [tx {:keys [cromwell items] :as _workload}]
   (let [env (@get-cromwell-wgs-environment cromwell)]
-    (->> items
-         (postgres/get-table tx)
-         (run! (partial maybe-update-workflow-status! tx env items)))))
+    (try
+      (let [workflows (postgres/get-table tx items)]
+        (run! (partial maybe-update-workflow-status! tx env items) workflows))
+      (catch ExceptionInfo (throw (ex-info "Error updating workload status" {:cause "no-workflows-found"}))))))
```

Comment:
What if the database is inaccessible? What if something goes wrong in `maybe-update-workflow-status!`? The `cause` might not be "no workflows found"
This message might be a red herring in the future - maybe we should wrap the underlying exception and add context:
```
(try
  (fetch-table-and-update)
  (catch Exception cause
    (throw (exec-info "Error updating worklflow status" {} cause))))